cli_tools/common/distro/distro.go

```diff
@@ -183,6 +183,17 @@ func newLinuxRelease(distro string, major string, minor string) (Release, error)
 	}
 }
 
+// FromGcloudOSArgumentMustParse parses the argument provided to the `--os` flag of
+// `gcloud compute images import`, and returns a Release if it represents a
+// release we *may* support. If osFlagValue does not parse, the call panics.
+func FromGcloudOSArgumentMustParse(osFlagValue string) Release {
```

Comment:
It seems this is a test utility function, not used for prod code. If so, it should probably live in a test/test util code, unless you expect it to be used in prod code in the future.

Reply:
It's useful for defining constants, similar to [1], and I envisioned it being used outside test code. That said I don't have an immediate usecase for it. Given all that, I'll leave it up to you --  I'm fine to move it, or leave it. 

1. https://golang.org/pkg/regexp/#MustCompile